Evtx/BinaryParser.py

```diff
@@ -186,7 +186,10 @@ def dosdate(dosdate, dostime):
 
 def parse_filetime(qword):
     # see http://integriography.wordpress.com/2010/01/16/using-phython-to-parse-and-present-windows-64-bit-timestamps/
-    return datetime.utcfromtimestamp(float(qword) * 1e-7 - 11644473600)
+    try:
+        return datetime.utcfromtimestamp(float(qword) * 1e-7 - 11644473600)
+    except ValueError:
+        return
```

Comment:
> But in the former case, I still care about the rest of the record.

Good point.

> Perhaps we should just return datetime.min?

I think this is a fair compromise. Lets try that out, and if it doesn't work, we can revisit the issue later on.